### PR TITLE
Add FDC information into pension form

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4006

Still have some tests to add, but this adds a lot of conditional helper text to the pensions form for FDC. It also makes the chapter progress bar hide chapter segments if there are no active pages in that chapter.

screen shot 2017-07-25 at 3 29 05 pm
screen shot 2017-07-25 at 3 28 58 pm
screen shot 2017-07-29 at 1 57 50 pm
screen shot 2017-07-29 at 1 57 46 pm
screen shot 2017-07-29 at 1 57 38 pm
screen shot 2017-07-29 at 1 56 23 pm
screen shot 2017-07-29 at 1 56 14 pm
screen shot 2017-07-29 at 1 55 49 pm
screen shot 2017-07-29 at 1 55 12 pm

### CODE REVIEW

File: src/js/pensions/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -52,10 +52,11 @@ class IntroductionPage extends React.Component {
               </div>
               <br/>
               <h6>Aid and Attendance and Housebound benefits</h6>
-              If you’re housebound, you may qualify for Aid and Attendance and Housebound benefits in addition to your pension benefits.
+              If you need help with your daily living activities, or if you’re confined to your home, you may qualify for an Aid and Attendance (A&A) or Housebound allowance in addition to your pension benefits.<br/>
+              <a href="/pension/aid-attendance-housebound/">Learn more about eligibility</a>.<br/><br/>
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: Should we open link in new tabs using target="_blank", like you've done  in `src/js/pensions/helpers.jsx` below?

**Review by cehsu — APPROVED** [review]
Looks good to me.

**Review by cvalarida — APPROVED** [review]

**Comment by goldenmeanie** [comment]
@bethpotts we'll need to address those CI failures, but you should be able to review this PR for content

**Comment by jbalboni** [comment]
@goldenmeanie CI errors were for broken links, should be fixed now.

File: content/pages/burials-and-memorials/survivor-and-dependent-benefits/burial-costs.md [hunk 2]
```diff
@@ -7,32 +7,10 @@ display_title: Handling Burial Costs
 
 <div class="va-introtext">
 
-If you’re the spouse of record (the legally recognized spouse) or a designated (legally chosen) family member of a Veteran who died, we may pay you a set amount for burial and funeral costs. This is called an allowance. You may also qualify for plot or burial allowances to help pay for the area of land used for the burial and other costs related to the burial. We may help with transportation costs as well. Find out how to get allowances.
```
Comment (jbalboni) [inline content/pages/burials-and-memorials/survivor-and-dependent-benefits/burial-costs.md]: @goldenmeanie Why is this page in here?

**Pull request merged**